README.md

```diff
@@ -253,7 +269,131 @@ Python | 3.6 or higher | [PSF](https://docs.python.org/3.6/license.html)
 
 2. Clone this project to `<path to onemkl>`, where `<path to onemkl>` is the root directory of this repository.
 
-3. [Build with CMake](#building-with-cmake).
+3. You can [Build with Conan](#building-with-conan) to automate the process of getting dependencies or you can download and install the required dependencies manually and [Build with CMake](#building-with-cmake) directly .
+
+*Note: Conan package manager automates the process of getting required packages, so that you do not have to go to different web location and follow different instructions to install them.*
+
+---
+
+## Building with Conan
+
+** Make sure you have completed [Build Setup](#build-setup). **
```

Comment:
I guess we need to specify somewhere that conan doesn't support Windows build so far

Reply:
Added a note.